packages/clay-card/src/Detail.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import React from 'react';
+
+import classNames from 'classnames';
+
+import Context from './Context';
+
+interface CardDetailProps extends React.HTMLAttributes<HTMLDivElement> {
+	children: React.ReactNode;
+}
+
+export type CardDetailType = React.FunctionComponent<CardDetailProps>;
```

Comment:
I'm seeing a bit of this pattern through the code, maybe it's best to keep this inside the function and if you need this type, you could use `typeof Detail` that typescript would take care of the rest.

Reply:
Yes, makes sense. Cleaning the repetitions.